.github/workflows/daily-work.yaml

```diff
@@ -0,0 +1,47 @@
+name: Daily work
+
+on:
+  workflow_run:
+    workflows: ['e2e tests']
+    types: [completed]
+
+jobs:
+  build-operator:
+    runs-on: ubuntu-latest 
+    if: ${{ github.event.workflow_run.conclusion == 'success' && contains('server daily version', github.event.workflow_run.display_title) }}  
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        path: vertica-kubernetes
+        fetch-depth: 0
+
+    - name: Save current version to env VERSION
+      shell: bash
+      run: |
+        cd vertica-kubernetes
+        echo $(make echo-versions | grep -e '^VERSION=' ) >> $GITHUB_ENV
+    - name: Save current sha to env VERSION_SHA
+      shell: bash
+      run: |
+        cd vertica-kubernetes
+        git tag --list
+        echo "VERSION_SHA=$(git rev-list -n 1 v$VERSION)" >> $GITHUB_ENV
+    - name: Download the release artifacts
+      run: |
+        cd vertica-kubernetes
+        scripts/download-release-artifacts.sh -d ci-artifacts $VERSION
+        find ci-artifacts
+      env:
+        GH_TOKEN: ${{ github.token }}
+
+    - name: Login to Docker Hub
+      uses: docker/login-action@v3
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - name: Upload new image
+      run: |
+        echo "Pushing images with tag: $VERSION_SHA"
+        # FIXME: uncomment the following
+        # docker buildx imagetools create --tag opentext/verticadb-operator:$VERSION ghcr.io/vertica/verticadb-operator:$VERSION_SHA
```

Comment:
Use the repo "opentext/verticadb-operator-private" to replace "opentext/verticadb-operator"

Reply:
Updated in https://github.com/vertica/vertica-kubernetes/pull/929/commits/cc05ac3845e2ed5b6da9410247a8d3b3751254e4